pkg/util/cgroups/cgroups_linux.go

```diff
@@ -0,0 +1,73 @@
+/*
```

Comment:
@timstclair will a sub-package of `cm` work? Having all cgroups logic in there will help with code structure.